crates/typst-library/src/layout/stack.rs

```diff
@@ -5,12 +5,11 @@ use crate::engine::Engine;
 use crate::foundations::{cast, elem, Content, NativeElement, Packed, Show, StyleChain};
 use crate::layout::{BlockElem, Dir, Spacing};
 
-/// Arranges content and spacing horizontally or vertically.
+/// コンテンツと間隔を垂直方向または水平方向に配置。
```

Comment:
「方向」という文字がダブるので、前方の方を省略してもよいかなと思いました。

```suggestion
/// コンテンツと間隔を垂直または水平方向に配置。
```